extension/chrome/settings/modules/my_key.htm

```diff
@@ -49,7 +49,7 @@
     </div>
     <div class="line"><span class="red_label">Never send this to anyone!</span></div>
     <div class="line">
-      <a href="#" class="action_download_prv bad">Save private key to a file</a>
+      <a href="#" class="action_download_prv bad" data-test="action-download-prv">Save private key to a file</a>
```

Comment:
@martgil we're using hyphenated ids for `data-test`, please notice.

Reply:
Thank you, sir. I'll try to be more mindful next time.